development-setup-wizard.py

```diff
@@ -0,0 +1,244 @@
+    run_tests(args)
+    finish(args)
+
+
+def check_prereqs(args):
+    if not shutil.which('docker'):
+        die(
+            'required command `docker` not found!',
+            'please make sure that you have it installed and try again')
+
+    hooray('Found all requirements')
+
+
+def create_env_files(args):
+    if os.environ.get('CAMPHORIC_SETUP_WIZARD_RECREATE_ENV') == 'no':
+        return
+
+    if args.preserve:
+        return
+
+    dir_name = './env'
+    if (os.path.isdir(dir_name)
+            and 'CAMPHORIC_SETUP_WIZARD_RECREATE_ENV' not in os.environ):
+        answer = input(' '.join([
+            f'it looks like "{dir_name}" already exists,'
+            'do you want to blow it away and start over? [y/n] ']))
+        if answer.lower() != 'y':
+            return
+
+        shutil.rmtree(dir_name)
+
+    shutil.copytree('./.env.example', dir_name)
+
+    hooray('Successfully created env files')
+
+
+def check_node_modules_folder(args):
+    dir_name = './client/node_modules'
+    contents = os.listdir(dir_name) if os.path.exists(dir_name) else []
+
+    if args.preserve:
+        return
+
+    if len(contents):
+        answer = input(' '.join([
+            f'it looks like "${dir_name}" is not empty,'
+            'do you want to blow it away and start over? [y/n] ']))
+        if answer.lower() == 'y':
+            shutil.rmtree(dir_name)
+
+    hooray(f'Checking of {dir_name} completed')
+
+
+def docker_compose_up(args):
+    print(bold('(Re)building docker containers'))
+    result = subprocess.run(['docker', 'compose', 'up', '--build', '-d'])
+    if result.returncode != 0:
+        die('failed to start docker containers')
+    hooray('Successfully built and started docker containers')
+
+
+def set_env_values(args):
+    if os.environ.get('CAMPHORIC_SETUP_WIZARD_RECREATE_SECRET_KEY') == 'no':
+        return
+
+    filename = './env/local/django.env'
+    contents = ''
+    with open(filename) as f:
+        contents = f.read().strip()
+
+    current_key_pair = ''
+    for line in re.split(r'[\r\n]+', contents):
+        if line.startswith('SECRET_KEY='):
+            current_key_pair = line
+            break
+
+    current_key = ''
+    if current_key_pair:
+        current_key = current_key_pair.split('=')[1]
+    else:
+        current_key = 'your-django-secret-key-here'
+        contents += '\nSECRET_KEY=your-django-secret-key-here'
+
+    if args.preserve:
+        return
+
+    if (current_key != 'your-django-secret-key-here'
+            and 'CAMPHORIC_SETUP_WIZARD_RECREATE_SECRET_KEY' not in os.environ):
+        answer = input(' '.join([
+            f'it looks like there is already a SECRET_KEY in your {filename} file.',
+            'Do you want replace it with a newly generated one? [y/n] ']))
+        if answer.lower() != 'y':
+            return
+
+    secret_key = get_secret_key()
+    contents = re.sub(r'SECRET_KEY=.*', f'SECRET_KEY={secret_key}', contents)
+    with open(filename, 'w') as f:
```

Comment:
"filename" and "f" are weak documenters.  Since you are introducing the variable "f", let's give it a more interesting name, perhaps "env_file".